x-pack/test/detection_engine_api_integration/security_and_spaces/rule_execution_logic/query.ts

```diff
@@ -237,51 +237,13 @@ export default ({ getService }: FtrProviderContext) => {
       expect(previewAlerts[0]?._source?.user?.risk).to.eql(undefined);
     });
 
-    describe('with host risk index', () => {
```

Comment:
Should we not have an equivalent test for the new data source?

Reply:
This test was specifically when we install host transforms and not host, also right now we have both

Reply:
But isn't this still valid if they don't have the new engine enabled? Until that code is gone, I think these should stay. Please correct me if I'm misunderstanding something.